docs/references/commands/cli/app/epinio_app_export.md

```diff
@@ -4,16 +4,21 @@ sidebar_label: "epinio app export"
 ---
 ## epinio app export
 
-Export the named application into the directory
+Export the named application into the directory or flag-specified registry
 
 ```
-epinio app export NAME DIRECTORY [flags]
+epinio app export [FLAGS] NAME [DIRECTORY] [flags]
```

Comment:
The double `flags` is an issue in the command `Use`: https://github.com/epinio/epinio/pull/2509#discussion_r1324458047